src/app/modules/main/chat_section/module.nim

```diff
@@ -399,4 +400,9 @@ method onContactBlocked*(self: Module, publicKey: string) =
 
 method onContactDetailsUpdated*(self: Module, publicKey: string) =
   let (chatName, chatImage, isIdenticon) = self.controller.getOneToOneChatNameAndImage(publicKey)
+  let contact = self.controller.getContactById(publicKey)
+  if (contact.hasAddedUs and not contact.added):
```

Comment:
@jrainville we need to check here if contact is not blocked as well, so please updated this condition to (like it's in `initContactRequestsModel` proc):
```
if(contact.requestReceived() and not contact.isContact() and not contact.isBlocked()):
```